feat: add page descriptions and update titleType (#340)
carbon-design-system/carbon-website: src/pages/components/button/code.mdx

---
title: Button

> description: Buttons express what action will occur when the user clicks or touches it. Buttons are used to initialize an action, either in the background or foreground of an experience.

Reviewer: ```suggestion
description: Buttons are used to initialize an action. Button labels express what action will occur when the user interacts with it.```
